comtypes/test/test_portabledevice.py

```diff
@@ -0,0 +1,146 @@
+import unittest as ut
+
+import comtypes
+import comtypes.client
+
+comtypes.client.GetModule("portabledeviceapi.dll")
+import comtypes.gen.PortableDeviceApiLib as port_api
+
+comtypes.client.GetModule("portabledevicetypes.dll")
+import comtypes.gen.PortableDeviceTypesLib as port_types
+
+
+def newGuid(*args: int) -> comtypes.GUID:
+    guid = comtypes.GUID()
+    guid.Data1 = c_uint32(args[0])
+    guid.Data2 = c_uint16(args[1])
+    guid.Data3 = c_uint16(args[2])
+    for i in range(8):
+        guid.Data4[i] = c_int8(args[3 + i])
+    return guid
+
+
+def PropertyKey(*args: int) -> _Pointer[port_api._tagpropertykey]:
+    assert len(args) == 12
+    assert all(isinstance(x, int) for x in args)
+    propkey = port_api._tagpropertykey()
+    propkey.fmtid = newGuid(*args[0:11])
+    propkey.pid = c_ulong(args[11])
+    return pointer(propkey)
+
+
+class Test_IPortableDevice(ut.TestCase):
+    # To avoid damaging or changing the environment, do not CREATE, DELETE or UPDATE!
+    # Do READ only!
+    def setUp(self):
+        info = comtypes.client.CreateObject(
+            port_types.PortableDeviceValues().IPersist_GetClassID(),
+            clsctx=comtypes.CLSCTX_INPROC_SERVER,
+            interface=port_types.IPortableDeviceValues,
+        )
+        mng = comtypes.client.CreateObject(
+            port_api.PortableDeviceManager().IPersist_GetClassID(),
+            clsctx=comtypes.CLSCTX_INPROC_SERVER,
+            interface=port_api.IPortableDeviceManager,
+        )
+        p_id_cnt = pointer(c_ulong())
+        mng.GetDevices(POINTER(c_wchar_p)(), p_id_cnt)
+        if p_id_cnt.contents.value == 0:
+            self.skipTest("There is no portable device in the environment.")
+        dev_ids = (c_wchar_p * p_id_cnt.contents.value)()
+        mng.GetDevices(cast(dev_ids, POINTER(c_wchar_p)), p_id_cnt)
+        self.device = comtypes.client.CreateObject(
+            port_api.PortableDevice().IPersist_GetClassID(),
+            clsctx=comtypes.CLSCTX_INPROC_SERVER,
+            interface=port_api.IPortableDevice,
+        )
+        self.device.Open(list(dev_ids)[0], info)
+
+    def test_EnumObjects(self):
+        WPD_OBJECT_NAME = PropertyKey(
+            0xEF6B490D, 0x5CD8, 0x437A, 0xAF, 0xFC, 0xDA, 0x8B, 0x60, 0xEE, 0x4A, 0x3C, 4)
+        WPD_OBJECT_CONTENT_TYPE = PropertyKey(
+            0xEF6B490D, 0x5CD8, 0x437A, 0xAF, 0xFC, 0xDA, 0x8B, 0x60, 0xEE, 0x4A, 0x3C, 7)
+        WPD_OBJECT_SIZE = PropertyKey(
+            0xEF6B490D, 0x5CD8, 0x437A, 0xAF, 0xFC, 0xDA, 0x8B, 0x60, 0xEE, 0x4A, 0x3C, 11)
+        WPD_OBJECT_ORIGINAL_FILE_NAME = PropertyKey(
+            0xEF6B490D, 0x5CD8, 0x437A, 0xAF, 0xFC, 0xDA, 0x8B, 0x60, 0xEE, 0x4A, 0x3C, 12)
+        WPD_OBJECT_PARENT_ID = PropertyKey(
+            0xEF6B490D, 0x5CD8, 0x437A, 0xAF, 0xFC, 0xDA, 0x8B, 0x60, 0xEE, 0x4A, 0x3C, 3)
+        folderType = newGuid(0x27E2E392, 0xA111, 0x48E0,
+                             0xAB, 0x0C, 0xE1, 0x77, 0x05, 0xA0, 0x5F, 0x85)
```

Comment:
```suggestion
        folder_type = comtypes.GUID("{27E2E392-A111-48E0-AB0C-E17705A05F85}")
```

It should be applied to other `newGuid`s.